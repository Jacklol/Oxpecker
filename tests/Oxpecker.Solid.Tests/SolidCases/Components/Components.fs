module Oxpecker.Solid.Tests.Cases.Components

open Oxpecker.Solid

[<SolidComponent>]
let Child () =
    h1() { "Hello" }

[<SolidComponent>]
let Parent () =
    __() {
        Child()
    }
