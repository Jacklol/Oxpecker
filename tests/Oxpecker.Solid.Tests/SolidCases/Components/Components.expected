
export function Child() {
    return <h1>
        Hello
    </h1>;
}

export function Parent() {
    return <>
        {Child()}
    </>;
}

